modules/payments/modules/payments/api.js

```diff
@@ -3,7 +3,7 @@ import { getGlobalOptions } from "@options";
 
 const global = getGlobalOptions(); 
 const BASE_URL = global.url; // change your BASE_URL in `options/options.js` to edit this value
-const token = "Token 676364403988909cde7f501aa2efeaf9ca30d18c"
+const token = "Token 913fdab66a6d73f3ed45a57a4fe6e8fa092fd153"
```

Comment:
Check my notes further below on how to access the token without having it exposed in the source code.

Reply:
As mentioned in the previous review, you should make use of the login token and mention in the README.md of this module that this module depends on the login module being installed and configured.

Reply:
we decided on improving the login module first to presist the token and store in encrypted storage and keep this as is for now.

Reply:
Yeah, recalling that now, sounds good

Reply:
But let's please replace it with a placeholder 
```suggestion
const token = "Your token here"
```